plugins/services/src/js/components/DeploymentsModal.js

```diff
@@ -37,7 +38,7 @@ const columnClasses = {
 };
 const columnHeadings = ResourceTableUtil.renderHeading({
   id: i18nMark("Affected Services"),
-  startTime: i18nMark("Started"),
+  version: i18nMark("Started"),
```

Comment:
Sorry, I think I'm missing something here - why does this key need to be changed to "version"?

Reply:
Same question from me. My concern is that it's not an accurate description of the column and could be confusing in the future. I tried changing this and line 195 to "startTime" and it seems to be ok?

Reply:
So debugging this it appears that `startTime` on `Deployment` is a `Date` object where are `version` is an ISO-8601 timestamp. So doing a naive comparison with `TableUtil.getSortFunction` (see line 181) on the `version` prop fixes this sorting issue, where as we would have to write a sort function to do comparisons on the Date object.

Reply:
@mperrotti , @natmegs , startTime of the object was undefined. Maybe it was a property that we used to have.